optimum/exporters/onnx/model_configs.py

```diff
@@ -261,6 +284,11 @@ def add_past_key_values(self, inputs_or_outputs: Dict[str, Dict[int, str]], dire
                 1: decoder_sequence_name,
             }
 
+    def patch_model_for_export(
+        self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
+    ) -> "ModelPatcher":
+        return BloomModelPatcher(self, model, model_kwargs=model_kwargs)
```

Comment:
Since this pattern is repeated multiple times, should we add a class attribute called `MODEL_PATCHER_CLASS`, and have one method implemented in the base class making use of it?

Reply:
yes let's add this in a following PR